test/integration_test/kubevirt_test.go

```diff
@@ -67,15 +85,12 @@ func kubevirtDeployFedoraVMWithClonePVC(t *testing.T) {
 	defer updateDashStats(t.Name(), &testResult)
 	instanceID := "vm"
 	appKey := "kubevirt-fedora"
+	var allCtxs []*scheduler.Context
 
-	ctxs := kubevirtVMsDeployAndValidate(
-		t,
-		instanceID,
-		[]string{appKey},
-	)
+	kubevirtVMScaledDeployAndValidate(t, instanceID, appKey, kubevirtScale, &allCtxs)
```

Comment:
Just curious. Why not add the scale param to the existing kubevirtVMsDeployAndValidate function instead of creating a new function? Also, why return the allCtxs C-style with pointer arg? Why not just return it as a return param which is more go-style?

Reply:
1. Added a new function instead of using the existing `kubevirtVMsDeployAndValidate` so that if future tests need to call this method, they can still use it w/o worrying about the scale factor
2. Agreed, fixed `allCtxs` to be a returned param from the function